src/components/Wizard/InfoBarWizard/InfoBarButtonsInput.tsx

```diff
@@ -0,0 +1,213 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import { useState } from "react";
+import Button from "react-bootstrap/Button";
+import Card from "react-bootstrap/Card";
+import Col from "react-bootstrap/Col";
+import Form from "react-bootstrap/Form";
+import Nav from "react-bootstrap/Nav";
+import Row from "react-bootstrap/Row";
+import Tab from "react-bootstrap/Tab";
+import { faPlus, faTrash } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { useFieldArray, useFormContext } from "react-hook-form";
+
+import { InfoBarWizardFormData } from "./formData";
+import LocalizableTextInput from "../../LocalizableTextInput";
+import {
+  RegisteredFormControl,
+  RegisteredFormCheck,
+} from "../../RegisteredFormControl";
+import { validateJsonAsObject } from "../validators";
+import ErrorMessage from "../../ErrorMessage";
+
+const defaults = (): InfoBarWizardFormData["content"]["buttons"][number] => ({
+  label: {
+    localized: false,
+    text: "",
+  },
+  accessKey: "",
+  primary: false,
+  supportPage: "",
+  action: "{}",
+});
+
+const controlPrefix = "content.buttons";
+
+export default function InfoBarButtonsInput() {
+  const { control, register } = useFormContext<InfoBarWizardFormData>();
+  const { fields, append, remove } = useFieldArray<InfoBarWizardFormData>({
+    control,
+    name: controlPrefix,
+  });
+  const [activeKey, setActiveKey] = useState<number | null>(
+    fields.length ? 0 : null
+  );
+
```

Comment:
If I'm understanding correctly, `setActiveKey` should be setting the active tab (allowing navigation between tabs?)

Reply:
Yes. By default it is set to the first tab (if any) or to null (if none).